NitroxClient/Helpers/NitroxEntityExtensions.cs

```diff
@@ -85,7 +86,8 @@ public static bool TryGetNitroxId(this Component component, out NitroxId nitroxI
         }
         if (!component.TryGetComponent(out NitroxEntity nitroxEntity))
         {
-            Log.Warn($"[{filePath[(filePath.LastIndexOf("\\", StringComparison.Ordinal) + 1)..^2] + methodName}():L{lineNumber}] Couldn't find an id on {component.GetFullHierarchyPath()}");
+            Action<string> warnMethod = warnOnce ? Log.WarnOnce : Log.Warn;
+            warnMethod.Invoke($"[{filePath[(filePath.LastIndexOf("\\", StringComparison.Ordinal) + 1)..^2] + methodName}():L{lineNumber}] Couldn't find an id on {component.GetFullHierarchyPath()}");
```

Comment:
I don't really like this method for what it does only give general context ("a certain gameobject happened not be marked with an id when it was required"). To keep it, we could just create a static cache in NitroxEntityExtensions with already printed GameObject paths to avoid repeating those warns (it would be absolutely useless to have this repeatedly logged).